packages/next-swc/crates/core/tests/fixture/relay/output.js

```diff
@@ -1,13 +1,10 @@
-const variableQuery = require('__generated__/InputVariableQuery.graphql.ts')
-fetchQuery(require('__generated__/InputUsedInFunctionCallQuery.graphql.ts'))
+const variableQuery = require("$DIR/__generated__/InputVariableQuery.graphql.ts");
```

Comment:
Where is this `$DIR` coming from? Is this expected?

Reply:
I think it is the result of `root_dir.join(...)`. Not really sure...